src/gui/widgets/TabWidget.cpp

```diff
@@ -274,10 +275,15 @@ void TabWidget::paintEvent( QPaintEvent * pe )
 			if( it.key() == m_activeTab )
 			{
 				p.fillRect( tab_x_offset, 2, ( *it ).nwidth - 6, m_tabbarHeight - 4, tabSelected() );
+				p.setPen(tabTextSelected());
+				p.drawText( tab_x_offset + 3, m_tabheight + 1, ( *it ).name );
+			}
+			else
+			{
+				// Draw text
+				p.setPen( tabText() );
```

Comment:
```suggestion
				p.setPen(tabText());
```

Reply:
Spaces fixed with commit 0fa18d405e7 where I have fixed style issues for all of `TabWidget`.